<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入长度超过6的串：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入子串：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="originalStr">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="aimSubStr">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="showSubStrButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>mid(2,3)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showLengthButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>length()</string>
   </property>
  </widget>
  <widget class="QPushButton" name="indexOfSubStrButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>indexOf(子串)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upperStrButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ToUpper()</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addSubStrButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+ 子串</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compareLenButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>原串 &gt; 子串 ?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addNum1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addNum2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QLabel" name="addResLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>res</string>
   </property>
  </widget>
  <widget class="QLabel" name="showSubStrLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>res</string>
   </property>
  </widget>
  <widget class="QLabel" name="showLengthLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>122</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>res</string>
   </property>
  </widget>
  <widget class="QLabel" name="indexOfSubStrLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>res</string>
   </property>
  </widget>
  <widget class="QLabel" name="upperStrLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>res</string>
   </property>
  </widget>
  <widget class="QLabel" name="addSubStrLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>res</string>
   </property>
  </widget>
  <widget class="QLabel" name="compareLenLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>res</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
